src/dm_logger.vala

```diff
@@ -199,7 +205,7 @@ namespace DMLogger
 
     public LogEntry.file_info(string filename, string git_version, int64 file_id, uint16 line, uint16 trace_level, bool concat)
```

Comment:
Lass dir bitte hier auch gleich die Komponente übergeben und gib sie dann weiter beim `this(...)` Aufruf.
Außerdem schreib bitte einen Valadoc Kommentar zum `file_info`-Konstruktor